src/Hl7.Fhir.Specification/Specification/Navigation/ElementNavigatorSlicingExtensions.cs

```diff
@@ -169,6 +170,7 @@ public static bool MoveToFirstReslice(this ElementDefinitionNavigator nav)
             return false;
         }
 
+#if CAN_THIS_OLD_CODE_BE_REMOVED
```

Comment:
I am not really a fan of this. Can we not make this internal function `FindMemberSlices(ElementDefinitionNavigator, bool)` deprecated? It is only used by `BucketFactory` and there you already use the new function `FindMemberSlices(ElementDefinitionNavigator)`

Reply:
Ok, I'll be brave and just remove it (it's internal after all).